kubernetes/argocd/stacks/common/metallb.yml

```diff
@@ -2,6 +2,10 @@
 apiVersion: v1
 kind: Namespace
 metadata:
+  labels:
```

Comment:
Can you add a one line comment here explaining why this is needed (for future reference)
Something like
`# Add security labels for extra L2 permissions` or something more coherent

Reply:
Added comments in e027194.